src/main/java/de/hhu/propra/sharingplatform/model/User.java

```diff
@@ -55,20 +55,10 @@
     @Value("${passwords.pepper}")
```

Comment:
Be careful at merge: this class has many changes in dev!

Reply:
Sure thing. We only killed the two relations to the payment. They are now handled by the contract rather than the user itself. We will not touch any of the newly made changes when resolving the conflict.